CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## CHANGE LOG
 
+### Version 1.3.0 (September 7, 2021)
+* Adds public methods for suspending/discarding & resuming InApp Notifications
```

Comment:
@william-ct change this to - suspending, discarding & resuming instead of suspending/discarding

Reply:
Done👍